Bio/PDB/DSSP.py

```diff
@@ -248,6 +253,9 @@ def __init__(self, model, pdb_file, dssp="dssp"):
         @param dssp: the dssp executable (ie. the argument to os.system)
         @type dssp: string
         """
```

Comment:
You have not explained the new ``acc_array`` argument in the docstring (which is written using RST markup), which should list the three values. The docstring should also include the reference and explanation currently only in the code comments.